### ISSUE TITLE
Editor(Quill) is not functional in production (add reproduction)

### ISSUE THREAD

**Issue by ga676005** [primefaces/primevue#2532]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primevue/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=110
```

**CodeSandbox Case (Bug Reports)**
cannot reproduce because it only happens on production. It shows a warning message on chrome `[Violation] Added synchronous DOM mutation listener to a 'DOMNodeInserted' event. Consider using MutationObserver to make the page more responsive.` when the Editor is added to the DOM. I have looked it up and It seems to be an issue from Quill.

**Current behavior**
The dropdown of Editor stop working after toggling by v-if or route change.

**Expected behavior**
The dropdown of Editor should always work.

**Minimal reproduction of the problem with instructions**
https://user-images.githubusercontent.com/51456572/167564684-ddb8e1f0-a192-43f9-91d2-963672758a03.mp4
1. toggle v-if=true of the editor
2. dropdown is clickable
3. toggle v-if=false and true again
4. dropdown is not clickable afterwards

* **Vue version:** 3.2.31

* **PrimeVue version:** 3.12.6

* **Browser:**  Chrome and Firefox show message in the console when I toggle v-if of the Editor.

Chrome 101.0.4951.54
`[Violation] Added synchronous DOM mutation listener to a 'DOMNodeInserted' event. Consider using MutationObserver to make the page more responsive.`

Edge 101.0.1210.39

Firefox Developer Edition 101.0b4
`已棄用 Mutation Events。請改用 MutationObserver。` 
which means Mutation Events are abandoned, use MutationObserver instead.

**Comment by tugcekucukoglu** [comment]
Thanks for the instructions but still I could not replicate the issue. In production [PrimeVue Editor](https://www.primefaces.org/primevue/editor) seems to working. How can I replicate the issue? If you could send us a codesandbox sample, we could see the problem exactly.

**Comment by ga676005** [comment]
Thank you for your reply.
I have made a minimal reproduction. It only happens in production mode, so I create a repo. 

https://user-images.githubusercontent.com/51456572/167750578-30c91a62-c2d4-473b-8e57-15cd54c9a860.mp4



1. git clone https://github.com/ga676005/primevue-editor-bug-reproduction
2. npm i
3. npm run preview
4. open the page
5. click the toggle button to show `Editor`, notice the dropdown is clickable
6. click the toggle button twice to hide and show `Editor` again, now dropdown is not clickable

If you open the console, you will see this warning every time the editor is showed
![image](https://user-images.githubusercontent.com/51456572/167750669-67a3673c-0ffa-422f-abed-5ac42e7b4de5.png)

If it runs in dev mode by `npm run dev`, the Editor function well.

**Comment by ga676005** [comment]
@tugcekucukoglu 
Hi, I have added the reproduction.

**Comment by tugcekucukoglu** [comment]
Hi,

I've cloned repo and  run the command `npm run preview`. The result is this:

<img width="428" alt="Screen Shot 2022-05-27 at 16 26 18" src="https://user-images.githubusercontent.com/32294991/170708478-991b8b4f-a408-4a18-a791-debf0cca4f58.png">


How can I replicate the issue?

**Comment by ga676005** [comment]
Sorry that I forgot one step. You need to `npm run build` first to be able to `npm run preview`.

**Comment by minzdrav** [comment]
Hi Guys
I have exactly same issue with editor. Any chances we can get fix soon or I should try another editor?

**Comment by martinszeltins** [comment]
@tugcekucukoglu I am also having this same issue. It seems to not re-initialize after being toggled with `v-if`. It starts to work again only after doing a full page refresh. I will attach a video below. Hope this gets fixed soon.

Screencat:
![editornotowrking](https://user-images.githubusercontent.com/34019878/182773054-4c05bac7-3828-4ef9-a6b4-b62d16156aaf.gif)

**Comment by MartijnVandecasteele** [comment]
To anyone looking for a quick fix:
It works fine if you set the toolbar prop as an array of items instead of using the ref.

For example:
`modules: {
  toolbar: ['bold', 'italic', 'underline']
}`

**Comment by mquintero0118** [comment]
I still have the same issue? did anyone solve this?

**Comment by martinszeltins** [comment]
> I still have the same issue? did anyone solve this?

@mquintero0118 I assume if you still have this issue and it's still `Open` on Github then it hasn't been solved yet.

**Comment by tugcekucukoglu** [comment]
Please check the latest version. If you think the problem still persists, please create a new issue with a reproducer.

Thanks in advanced.
